api/app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,46 @@
+class AppointmentsController < ApplicationController
+    before_action :set_advisor, :set_user
+
+    def index
+        unless authorized
+            render json: { error: "Requester is not an advisor" }, status: :bad_request
+            return
+        end
+
+        @appointments = Appointment.where(user_id: @advisor.id, seen: FALSE).where('appointment_time >= ? ', Date.today).order('appointment_time ASC').limit(10)
```

Comment:
Why have the limit here? Will this be paginated?

Reply:
Definitely want to add pagination down the line, but didn't want to send too many appointments over in one request. Figured that if they can see the next X, those are the most important. Open to ideas here!

Reply:
If we're going to eventually add pagination then I think this is fine 👍